main.go

```diff
@@ -26,24 +28,26 @@ func main() {
 	// Create a new Discord session using the provided bot token.
 	dg, err := discordgo.New("Bot " + Token)
 	if err != nil {
-		fmt.Println("error creating Discord session,", err)
+		log.Fatal("Error creating Discord session", err)
 		return
```

Comment:
You don't need of `return` statement here. Cause, `log.Fatal` function emit a `os.Exit(1)`

Reply:
Please check: https://golang.org/src/log/log.go?s=10278:10306#L323

Reply:
Perfect!